tools/microbit-oob/main.ts

```diff
@@ -308,6 +309,12 @@ function processSensorGetReading(serviceClass: number, pkt: jacdac.JDPacket) {
             led.plotBarGraph(position, 100)
             setPixelBrightness(position / 200) // don't go above 50%
         }
+    } else if (serviceClass === jacdac.SRV_ILLUMINANCE) {
+        const illuminance = Math.round(pkt.jdunpack<[number]>("u22.10")[0])
+        if (illuminance !== sensorMap[lookup]) {
+            sensorMap[lookup] = illuminance
+            led.plotBarGraph(illuminance, 100)
```

Comment:
what is the range of illuminance?

Reply:
100ish in my office to 100k outside on sunny day. I guess we should plot log of luminance? 

BTW this PR is really a big report - it gives 845 on hw